### PR TITLE
animation start in when visible

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

File: src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
 <ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
-        xmlns:g="urn:import:com.google.gwt.user.client.ui"
-        xmlns:slider="urn:import:com.arcbees.website.client.application.slider"
-        xmlns:seo="urn:import:com.arcbees.seo.widget"
-        ui:generateFormat='com.google.gwt.i18n.rebind.format.PropertiesFormat'
-        ui:generateKeys="com.google.gwt.i18n.server.keygen.MD5KeyGenerator"
-        ui:generateLocales="default">
+             xmlns:g="urn:import:com.google.gwt.user.client.ui"
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml]: Specific to intelliJ, didn't we drop that code style?

File: src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml [hunk 2]
```diff
@@ -167,46 +167,88 @@
...
+                                            <g:Image resource="{page.trackR}"/>
+                                        </a>
+                                    </li>
+                                </ul>
+                            </g:HTMLPanel>
+                        </slider:content>
+                    </slider:Slider>
+                </div>
+
+                <div class="{res.style.mobileVisible}">
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml]: Why not applying the style directly on the slider and remove that div?

File: src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml [hunk 3]
```diff
@@ -167,46 +167,88 @@
                 <h2>
                     <ui:msg description="Professional Services - Trusted By">Trusted By</ui:msg>
                 </h2>
-                <slider:Slider>
-                    <slider:content>
-                        <g:HTMLPanel>
-                            <ul>
-                                <li>
+                <div class="{res.style.mobileHidden}">
```
Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml]: @LarryMatte All you change between Mobile and Desktop slider is that one is using `<ul><li>` and the other one `<div>` ... Could you keep the desktop one and change `<li>` to `display: block` instead with media-queries once on mobile version? That way, you wouldn't need to duplicate the slider and the data in it

I also think that the grid should have general classes for "mobile hidden" "mobile only" and all, as we will eventually define the breakpoints in the grid settings. But that's another story.

File: src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml (hunk 2 above)
Comment (LarryMatte) [inline src/main/java/com/arcbees/website/client/application/services/ServicesView.ui.xml]: When I apply the addStyleNames on the slider, the build is failing, I'll ask sp to check that out.

**Comment by jasonlemay** [comment]
LGTM

**Pull request merged**